sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-api/src/main/java/org/apache/shardingsphere/orchestration/center/yaml/swapper/InstanceConfigurationYamlSwapper.java

```diff
@@ -34,13 +34,13 @@
      */
     @Override
     public YamlInstanceConfiguration swap(final InstanceConfiguration data) {
-        YamlInstanceConfiguration result = new YamlInstanceConfiguration();
-        result.setOrchestrationType(data.getOrchestrationType());
-        result.setInstanceType(data.getType());
-        result.setServerLists(data.getServerLists());
-        result.setNamespace(data.getNamespace());
-        result.setProps(data.getProperties());
-        return result;
+        YamlInstanceConfiguration yamlInstanceConfiguration = new YamlInstanceConfiguration();
+        yamlInstanceConfiguration.setOrchestrationType(data.getOrchestrationType());
+        yamlInstanceConfiguration.setInstanceType(data.getType());
+        yamlInstanceConfiguration.setServerLists(data.getServerLists());
+        yamlInstanceConfiguration.setNamespace(data.getNamespace());
+        yamlInstanceConfiguration.setProps(data.getProperties());
+        return yamlInstanceConfiguration;
```

Comment:
The result variable need to rename as result

Reply:
> The result variable need to rename as result

why use result as variable name？

Reply:
FIY: https://shardingsphere.apache.org/community/en/contribute/code-conduct/

`Return values are named with result; Variables in the loop structure are named with each; Replace each with entry in map.`